src/Symfony/Component/Translation/Tests/Command/XliffLintCommandTest.php

```diff
@@ -129,6 +130,48 @@ public function testGetHelp()
         $this->assertStringContainsString($expected, $command->getHelp());
     }
 
+    public function testLintIncorrectFileWithGithubFormat()
+    {
+        if (!class_exists(GithubActionReporter::class)) {
```

Comment:
This tests looks weird, maybe you can add a test and use ClassMock and mock that it exists and does not